packages/core-commands/src/admin-navigation-commands.js

```diff
@@ -58,6 +59,5 @@ export function useAdminNavigationCommands() {
 				document.location.href = 'edit.php?post_type=wp_block';
 			}
 		},
-		icon: isSiteEditor ? symbol : external,
```

Comment:
Minor enhancement: I think Patterns page should not be considered External, whether you are inside or outside the Site Editor.